cdp/wallet.py

```diff
@@ -286,6 +288,23 @@ def create_address(self) -> "WalletAddress":
 
         return wallet_address
 
+    def create_webhook(self, notification_uri: str) -> "Webhook":
+        """Create a new webhook for the wallet.
+
```

Comment:
please add `Args` doc string section.

Reply:
Please document what the webhook is for. It is unclear to me what `wallet.creat_webhook()` is creating a webhook for. This is not a problem specifically with Python, but should be evaluated across all SDKs.

Reply:
modified, also added a section in `README` with more information .